packages/storybook/stories/components/card/documentation.mdx

```diff
@@ -124,9 +124,11 @@ All these elements are not mandatory in every **Card**.
 
 N/A
 
-## Variants
+## Variation
 
-### Colors
+Card has one variant that changes its border, **Primary**.
```

Comment:
Don't you mean something like the following?

"Card has one variation of color for its border".

### Color
**Neutral** ...
**Primary** ...